### PR TITLE
Docs: update docs to reflect new sharing UI

### LINKED ISSUE #88783: Update Grafana docs
As discussed [here](https://docs.google.com/document/d/1X073gsQ6RIoroVUGfT_4WdH7lO1OskSdgd2oelNehcs/edit), "Public Dashboards" is not a concept anymore.
We need to update the Grafana Documentation

- [ ] Change [panel shortcuts documentation](https://grafana.com/docs/grafana/latest/dashboards/use-dashboards/#keyboard-shortcuts) 
- [ ] Change [Public dashboard documentation ](https://grafana.com/docs/grafana/latest/dashboards/dashboard-public/)
- [ ] Change [share dashboard documentation](https://grafana.com/docs/grafana/latest/dashboards/share-dashboards-panels/)
- [ ] Change [Use dashboards](https://grafana.com/docs/grafana/latest/dashboards/use-dashboards/#dashboard-feature-overview)

### LINKED ISSUE #89064: Snapshots: Clarify publishing paths
### What happened?

The documentation here about publishing a snapshot is here https://grafana.com/docs/grafana/latest/dashboards/share-dashboards-panels/#publish-a-snapshot

It talks about two paths to publish, either to your local instance or to snapshots.raintank.io . The latter will allow you to publish a snapshot that can be accessed by anyone with the link.

Users commonly use custom panels to display data, and if a snapshot containing a custom panel is published to snapshot.raintank.io, if the plugin is not available, the snapshot receives an error. There have been instances historically where someone would upload the custom plugin to the snapshot server. This is not a path we want to support, and we should clarify this to people who come across this scenario.

### What did you expect to happen?

For an explanation, in either or both the documentation or in the app, describing the limitations on publishing to snapshots.raintank.io and that we do not allow uploading custom panels to that server.

### Did this work before?

A [forum post](https://community.grafana.com/t/snapshot-created-using-raintank-does-not-show-custom-panel-plugin/35772) indicates that it was possible to request upload of custom panels to that server historically.

### PR DESCRIPTION
**Pull request by imatwawana**
**TO DO**
- [x] Make updates based on action items from feature testing

**Merge this PR BEFORE PR #92957** 

This PR:

1. Updates the following pages significantly to reflect new UI: 
    - [Share dashboards and panels](https://grafana.com/docs/grafana/latest/dashboards/share-dashboards-panels/)
    - [Shared dashboards](https://grafana.com/docs/grafana/latest/dashboards/dashboard-public/)
    - [Manage library panels](https://grafana.com/docs/grafana/next/dashboards/build-dashboards/manage-library-panels/)
    - [Use dashboards](https://grafana.com/docs/grafana/latest/dashboards/use-dashboards/)
2. Renames the page **Public dashboards** (filename: `dashboard-public.md`) to **Shared dashboards** (filename: `shared-dashboards.md`)
3. Updates several other pages to replace "public dashboards" with "shared dashboards" terminology.

### CODE REVIEW

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Grafana enables you to share dashboards and panels with other users within an or
 - A Snapshot
 - An embedded link (for panels only)
 - An export link (for dashboards only)
+- A report
+
+<!-- has any of the information in these next 3 sentences changed? -->
```
Comment (lucychen-grafana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:59]: We have not touch permissions so these next three sentences should still stand as is

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 2]
```diff
@@ -54,6 +54,9 @@ Grafana enables you to share dashboards and panels with other users within an or
 - A Snapshot
 - An embedded link (for panels only)
 - An export link (for dashboards only)
+- A report
+
```
Comment (lucychen-grafana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:58]: Is this list just for the new share button, not inclusive of the new export button?

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 3]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
-{{< admonition type="note" >}}
-If you change a dashboard, ensure that you save the changes before sharing.
-{{< /admonition >}}
+- Internally with a link
+- Externally with a link (to anyone)
+- Externally by email (to specific people)
+- As a snapshot
+- As a report
+- As a JSON file
+
```
Comment (lucychen-grafana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:79]: Should we include "As a PDF" as well? The changes are in review for that

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 2 above)
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:58]: For now, yeah. That may change, but I want to see how the panel sharing looks before I finalize that intro text.

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 3 above)
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:79]: Will this be behind the newDashboardShareComponent feature flag as well?

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 4]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
+1. Navigate to **Dashboards > Snapshots** in the main menu.
+1. Click the red **x** next to the snapshot that you want to delete.
 
-You can generate and save PDF files of any dashboard.
+The snapshot is immediately deleted. You may need to clear your browser cache or use a private or incognito browser to confirm this.
 
-{{< admonition type="note" >}}
-Available in [Grafana Enterprise](ref:grafana-enterprise) and [Grafana Cloud](/docs/grafana-cloud/).
-{{< /admonition >}}
+### Export a dashboard as JSON
```
Comment (lucychen-grafana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:200]: Will need to include Export dashboard as PDF section too (before JSON since the button will appear first on the export menu)

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 5]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
-The dashboard export action creates a Grafana JSON file that contains everything you need, including layout, variables, styles, data sources, queries, and so on, so that you can later import the dashboard.
+### Share a snapshot
 
-1. Click **Dashboards** in the main menu.
-1. Open the dashboard you want to export.
-1. Click **Share** in the top-right corner.
-1. Click **Export**.
+A dashboard snapshot publicly shares a dashboard while removing sensitive data such as queries and panel links, leaving only visible metrics and series names. Anyone with the link can access the snapshot.
+
+<!-- is the following para still true? -->
```
Comment (lucychen-grafana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:169]: Yes, we still support the following at this moment. We do have a huge [epic](https://github.com/grafana/grafana/issues/75241) to improve snapshots in the future whenever we do prioritize it which will probably change snapshots a lot TBD

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 6]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share externally**.
+1. In the **Link access** drop-down, select **Anyone with the link**.
+1. Click the checkbox confirming that you understand the entire dashboard will be public.
+1. Click **Accept**.
+1. (Optional) Set the following options:
+   - **Enable time range** - Allow people accessing the link to change the time range. This configuration screen shows the default time range of the dashboard.
+   - **Display annotations** - Allow people accessing the link to view the dashboard annotations.
+1. Each recipient will receive an email with link to the dashboard that's valid for 30 days.
```
Comment (juanicabanas) [inline docs/sources/dashboards/share-dashboards-panels/index.md:135]: this part is not ok. there's no email or valid dashboard URL for 30 days. Anyone with the URL (that has to be copy pasted) will have access to the dashboard, until the user decides to "revoke access" (delete the public dashboard)

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 7]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share externally**.
+1. Do one of the following:
+   - Click **Pause access** so that people can't access the dashboard, but the link is maintained.
+   - Click **Resume access** so that people can access the dashboard again.
+   - Click **Revoke access** so that people can't access the dashboard unless a new external link is generated. Confirm that you want to revoke the link.
+1. Click the **X** at the top-right corner to close the share drawer.
 
-#### Delete a snapshot
+### Schedule a report
```
Comment (evictorero) [inline docs/sources/dashboards/share-dashboards-panels/index.md:151]: This is a paid feature. (just in case)

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 8]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
 
-1. Click **Dashboards** in the main menu.
-1. Open the dashboard you want to export.
-1. Click **Share** in the top-right corner.
-1. Click **Export**.
+A dashboard snapshot publicly shares a dashboard while removing sensitive data such as queries and panel links, leaving only visible metrics and series names. Anyone with the link can access the snapshot.
+
+<!-- is the following para still true? -->
+
+You can publish snapshots to your local instance or to [snapshots.raintank.io](http://snapshots.raintank.io). The latter is a free service provided by Grafana Labs that enables you to publish dashboard snapshots to an external Grafana instance. Anyone with the link can view it. You can set an expiration time if you want the snapshot removed after a certain time period.
```
Comment (evictorero) [inline docs/sources/dashboards/share-dashboards-panels/index.md:171]: The snapshots.raintank.io option is disabled by default in Cloud. Maybe we should mention the config that enable/disable it here.

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 9]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
+- Externally by email (to specific people)
+- As a snapshot
+- As a report
+- As a JSON file
+
+When you share a dashboard externally as a link or by email, those dashboards are included a list of your public dashboards. To view the list and manage these dashboards, navigate to **Dashboards > Public dashboards**.
+
+<!-- image of list here -->
+
+### Share internally
```
Comment (evictorero) [inline docs/sources/dashboards/share-dashboards-panels/index.md:84]: There is another feature not mentioned here, when clicking the Share button directly (not the dropdown) a dashboard link is copied to the clipboard with the options defined below.

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 10]
```diff
@@ -54,6 +54,9 @@ Grafana enables you to share dashboards and panels with other users within an or
 - A Snapshot
```
Comment (evictorero) [inline docs/sources/dashboards/share-dashboards-panels/index.md:54]: nit: I saw below we are not capitalizing the first letter on snapshots. Should we lowercase it here?

Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:54]:
```suggestion
- A snapshot
```

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 1 above)
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:59]:
```suggestion
```

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 11]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
 As of Grafana 8.0, anonymous access permission is not available in Grafana Cloud.
 {{< /admonition >}}
 
-When you share a panel or dashboard as a snapshot, a snapshot (which is a panel or dashboard at the moment you take the snapshot) is publicly available on the web. Anyone with a link to it can access it. Because snapshots do not require any authorization to view, Grafana removes information related to the account it came from, as well as any sensitive data from the snapshot.
-
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:67]: It's been reworded and moved down to line 167 in the Share a snapshot section

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 12]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
 ## Share a dashboard
 
-You can share a dashboard as a direct link or as a snapshot. You can also export a dashboard.
+You can share dashboards in the following ways:
 
-{{< admonition type="note" >}}
-If you change a dashboard, ensure that you save the changes before sharing.
-{{< /admonition >}}
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:74]: This seemed fairly obvious to me, but if you think this is something users are having an issue with, I can add it back in.

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 13]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
+1. (Optional) Set the following options (they're enabled by default):
+   - **Lock time range** - Change the current relative time range to an absolute time range.
+   - **Shorten link** - Shorten the dashboard link.
+1. Select the **Current**, **Dark**, or **Light** theme for the dashboard.
+1. Click **Copy link**.
+1. Send the copied link to a Grafana user with authorization to view the link.
+1. Click the **X** at the top-right corner to close the share drawer.
 
-### Share a direct link
+### Share externally with specific people
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:99]: The order of these in the body aligns with the UI, so I'll change the list at the top to match.

And I'm adding in a note about the availability here.

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 14]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
+You can share dashboards in the following ways:
 
-{{< admonition type="note" >}}
-If you change a dashboard, ensure that you save the changes before sharing.
-{{< /admonition >}}
+- Internally with a link
+- Externally with a link (to anyone)
+- Externally by email (to specific people)
+- As a snapshot
+- As a report
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:77]: I'll add a note

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 3 above)
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:79]:
```suggestion
- As a PDF

```

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 15]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
+1. Select the **Current**, **Dark**, or **Light** theme for the dashboard.
+1. Click **Copy link**.
+1. Send the copied link to a Grafana user with authorization to view the link.
+1. Click the **X** at the top-right corner to close the share drawer.
 
-### Share a direct link
+### Share externally with specific people
 
-The **Link** tab shows the current time range, template variables, and the default theme. You can also share a shortened URL.
+To share with specific external users, you can send them a link by email. Use this option when you only want to share your dashboard with specific people instead of anyone who navigates to the link. Sharing a link by email creates a 30-day token associated with that external user.
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:101]: Do you mean if I set up a link with your email address and you give that link to Eze without him accessing your email, the link will work?

Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:101]:
```suggestion
To share with specific external users, you can send them a link by email. Use this option when you only want to share your dashboard with specific people instead of anyone who navigates to the link. Sharing a link by email creates a 30-day token.
```

File: docs/sources/dashboards/share-dashboards-panels/index.md [hunk 16]
```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
...
 
-1. Click **Copy**.
+When you share a dashboard with an email link, your organization is billed per user for the duration of the 30-day token, regardless of how many dashboards are shared. Billing stops after 30 days unless you renew the token.
```
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:103]: I took this directly from the UI text so if this is wrong, then I think the UI is wrong too?

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 6 above)
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:135]:
```suggestion
1. Each recipient will receive an email with link to the dashboard.
```

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 7 above)
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:151]:
```suggestion
### Schedule a report

{{< admonition type="note" >}}
This feature is only available in Grafana Enterprise.
{{< /admonition >}}
```

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 5 above)
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/index.md:169]:
```suggestion
```

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 8 above)
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:178]: Happy to add it in or link to it. But what are the steps? I'm not seeing an obvious feature toggle.

Comment (evictorero) [reply docs/sources/dashboards/share-dashboards-panels/index.md:185]: It's inside the [configuration file](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#external_enabled) (not a feature toggle). I'm not saying we should remove this, but if a Cloud user reads this, they won't find these options since they are disabled in the Cloud. I'm not sure how to properly communicate this.

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (evictorero) [reply docs/sources/dashboards/share-dashboards-panels/index.md:115]: cc: @juanicabanas @natydej

File: docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md [hunk 17]
```diff
@@ -0,0 +1,310 @@
+---
+aliases:
+  - ../features/dashboard/dashboards/
+  - dashboard-manage/
```
Comment (jdbaldry) [inline docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md:4]:
```suggestion
  - ../dashboard-public/ # /docs/grafana/latest/dashboards/dashboard-public/
```

**Review by irenerl24 — APPROVED** [review]
LGTM! Thank you so much!

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md:74]: I've added it back in.

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md]: This is updated now.

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md]: Updated in the docs.

File: docs/sources/dashboards/share-dashboards-panels/index.md (hunk 8 above)
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/index.md]: Updated!

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 18]
```diff
@@ -0,0 +1,372 @@
...
+| scale     | Numeric value to configure device scale factor. Default is 1. Use a higher value to produce more detailed images (higher DPI). Supported in Grafana v7.0+. |
+
+You can also update these parameters in the [image rendering configuration](ref:image-rendering-config).
+
+The following example shows a link to a server-side rendered PNG:
+
+```bash
+https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&from=1568719680173&to=1568726880174&panelId=4&fullscreen
+```
+
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:277]: The Grafana Play link above wasn't valid anymore so I put in a new one, but when I generate a new one, the URL is suuuuper long and the Shorten link option doesn't help. Is there a way to make these links shorter?

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 19]
```diff
@@ -0,0 +1,372 @@
...
+To create a panel that can be embedded, follow these steps:
+
+1. Hover over any part of the panel you want to share to display the actions menu on the top-right corner.
+1. Click the menu and select **Share embed**.
+
+   The **Share embed** drawer opens.
+
+1. (Optional) Toggle the **Lock time range** switch to set whether the panel uses the current relative time range or an absolute time range. This option is enabled by default.
+1. Select the theme for the dashboard. Choose from **Current**, **Dark**, or **Light**.
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:296]: Can you make changes to the HTML at this step?

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 20]
```diff
@@ -0,0 +1,372 @@
...
+1. Navigate to **Dashboards > Snapshots** in the main menu.
+1. To confirm which snapshot you're about to delete, click **View** on the snapshot row.
+
+   The URLs for panel and dashboard snapshots from the same dashboard look similar and viewing them first can help you distinguish them.
+
+1. Click the red **x** next to the snapshot URL that you want to delete.
+
+The snapshot is immediately deleted. You may need to clear your browser cache or use a private or incognito browser to confirm this.
+
+### Share a library panel
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:357]: This isn't in the Share options, so should this even be on this page anymore? I think we either take it out entirely or at least remove the text and link folks to this page:
https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/manage-library-panels/

File: docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md [hunk 21]
```diff
@@ -0,0 +1,327 @@
...
+
+You can see a list of all your externally shared dashboards in one place by navigating to **Dashboards > Shared dashboards**. For each dashboard in the list, the page displays:
+
+- Link to view the externally shared version of the dashboard
+- Link to the shared dashboard configuration
+- Options to pause or revoke access to the external dashboard
+
+You can also click the name of the dashboard to navigate to the dashboard internally.
+
+## Security implications of sharing your dashboard externally
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md:64]: I know the word "implication" doesn't mean just risks, but I definitely expected to see more risks in this section vs things that seem actually pretty positive from a security perspective. Suggest something less scary.
```suggestion
## Important notes about sharing your dashboard externally
```

**Review by evictorero — APPROVED** [review]
LGTM!

File: docs/sources/dashboards/share-dashboards-panels/_index.md (hunk 18 above)
Comment (evictorero) [reply docs/sources/dashboards/share-dashboards-panels/_index.md:275]: Why the shorter link doesn't help? I've generated [this one](https://play.grafana.org/goto/LDBW8z6SR?orgId=1). Also, is it not possible to hide the link in the markup?

File: docs/sources/dashboards/share-dashboards-panels/_index.md (hunk 20 above)
Comment (evictorero) [reply docs/sources/dashboards/share-dashboards-panels/_index.md:352]: We can delete it since it is not part of the Share options anymore

File: docs/sources/dashboards/share-dashboards-panels/_index.md (hunk 18 above)
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/_index.md:275]: It looks like the idea was to actually show what the HTML link looks like rather than embedding it in text.

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 22]
```diff
@@ -0,0 +1,365 @@
...
+| tz        | Timezone in the format `UTC%2BHH%3AMM` where HH and MM are offset in hours and minutes after UTC.                                                          |
+| timeout   | Number of seconds. The timeout can be increased if the query for the panel needs more than the default 30 seconds.                                         |
+| scale     | Numeric value to configure device scale factor. Default is 1. Use a higher value to produce more detailed images (higher DPI). Supported in Grafana v7.0+. |
+
+You can also update these parameters in the [image rendering configuration](ref:image-rendering-config).
+
+The following example shows a link to a server-side rendered PNG:
+
+```bash
+https://play.grafana.org/render/d-solo/ktMs4D6Mk?from=2024-09-03T11:55:44.442Z&to=2024-09-03T17:55:44.442Z&panelId=panel-13&__feature.dashboardSceneSolo&width=1000&height=500&tz=UTC
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:273]:
```suggestion
https://play.grafana.org/d/000000012/grafana-play-home?from=2019-09-17T11:28:00.173Z&orgId=1&to=2019-09-17T13:28:00.174Z&timezone=browser
```

File: docs/sources/dashboards/share-dashboards-panels/_index.md (hunk 18 above)
Comment (imatwawana) [reply docs/sources/dashboards/share-dashboards-panels/_index.md:275]: The link you provided seems to just be a dashboard link though. This is meant to be a link for a panel rendered as an image.

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 23]
```diff
@@ -0,0 +1,365 @@
...
+1. Click the **X** at the top-right corner to close the share drawer.
+
+## Share panels {#share-a-panel}
+
+You can share a panels in the following ways:
+
+- [Internally with a link](#share-an-internal-link)
+- [As an embed](#share-an-embed)
+- [As a snapshot](#panel-snapshot)
+- [As a library panel](#share-a-library-panel)
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:234]:
```suggestion
```

File: docs/sources/dashboards/share-dashboards-panels/_index.md (hunk 18 above)
Comment (evictorero) [reply docs/sources/dashboards/share-dashboards-panels/_index.md:274]: You are right; no, we don't support shortened links for rendering panels.

File: docs/sources/dashboards/share-dashboards-panels/_index.md (hunk 19 above)
Comment (evictorero) [reply docs/sources/dashboards/share-dashboards-panels/_index.md:291]: Yes, the user can make changes in the editor and then copy the result.

File: docs/sources/dashboards/share-dashboards-panels/_index.md
Comment (juanicabanas) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:1]: - I think there's an error in this sentence: 
`Once you've a customized an internal link, you can share it quickly by following these steps:`

Should it be 
`Once you've customized an internal link, you can share it quickly by following these steps:`
?

- Is it worth mentioning that the configuration is stored in a browser scope? 
It says:
`This link has any customizations, like time range locking or theme, that you've previously set.` but it doesn't indicate that

File: docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md [hunk 24]
```diff
@@ -0,0 +1,323 @@
...
+1. Click the checkbox confirming that you understand the entire dashboard will be public.
+1. Click **Accept**.
+1. (Optional) Set the following options:
+   - **Enable time range** - Allow people accessing the link to change the time range. This configuration screen shows the default time range of the dashboard.
+   - **Display annotations** - Allow people accessing the link to view the dashboard annotations.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+Now anyone with the link can access the dashboard until you pause or revoke access to it.
+
+Once you've shared a dashboard externally, a **Public** label is displayed in the header of the dashboard.
```
Comment (juanicabanas) [inline docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md:176]: this also applies for email sharing

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 25]
```diff
@@ -26,7 +26,7 @@ While an image is being rendered, the PNG image is temporarily written to the fi
 
 A background job runs every 10 minutes and removes temporary images. You can configure how long an image should be stored before being removed by configuring the [temp_data_lifetime]({{< relref "../configure-grafana#temp_data_lifetime" >}}) setting.
 
-You can also render a PNG by hovering over the panel to display the actions menu in the top right corner, and then clicking **Share > Direct link rendered image** in the Link tab.
+You can also render a PNG by hovering over the panel to display the actions menu in the top-right corner, and then clicking **Share > Share link**. When you share an internal panel link, you also have the option to share it as an image.
```
Comment (juanicabanas) [inline docs/sources/setup-grafana/image-rendering/_index.md:29]: The sentence starts talking about rendering a PNG. Does it make sense to say 
`When you share an internal panel link, you also have the option to share it as an image.`? 
Should it be something like `and then clicking **Share > Share link**. Render image .`

**Review by juanicabanas — APPROVED** [review]
Great work 🙌🏻
I just left some questions/comments, but LGTM

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 26]
```diff
@@ -0,0 +1,348 @@
...
+   - **Lock time range** - Change the current relative time range to an absolute time range. This option is enabled by default.
+   - **Shorten link** - Shorten the dashboard link. This option is enabled by default.
+1. Select the theme for the dashboard. Choose from **Current**, **Dark**, or **Light**.
+1. Click **Copy link**.
+1. Send the copied link to a Grafana user with authorization to view the link.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+#### Quick-share an internal link
+
+Once you've a customized an internal link, you can share it quickly by following these steps:
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:116]:
```suggestion
Once you've customized an internal link, you can share it quickly by following these steps:
```

File: docs/sources/dashboards/share-dashboards-panels/_index.md [hunk 27]
```diff
@@ -0,0 +1,348 @@
...
+
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** button, not the drop-down list icon, to copy a shortened link.
+
+This link has any customizations, like time range locking or theme, that you've previously set.
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/_index.md:122]:
```suggestion
This link has any customizations, like time range locking or theme, that you've previously set. These are stored in the browser scope.
```

File: docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md [hunk 28]
```diff
@@ -0,0 +1,323 @@
...
+1. In the **Invite** field, enter the email address of the person you want to invite and click **Invite** and repeat this process for all the people you want to invite.
+
+   You can only invite one person at a time.
+
+1. (Optional) Set the following options:
+   - **Enable time range** - Allow people accessing the link to change the time range. This configuration screen shows the default time range of the dashboard.
+   - **Display annotations** - Allow people accessing the link to view the dashboard annotations.
+1. (Optional) Click **Copy external link** and send the copied URL to any external user.
+1. Click the **X** at the top-right corner to close the share drawer.
+
```
Comment (imatwawana) [inline docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md:98]:
```suggestion

Once you've shared a dashboard externally, a **Public** label is displayed in the header of the dashboard.

```

File: docs/sources/setup-grafana/image-rendering/_index.md (hunk 25 above)
Comment (imatwawana) [reply docs/sources/setup-grafana/image-rendering/_index.md:29]: I think I didn't phrase it well. Let me try again.
```suggestion
You can also render a PNG by hovering over the panel to display the actions menu in the top-right corner, and then clicking **Share > Share link**. The **Render image** option is displayed in the link settings.
```

Comment (juanicabanas) [reply docs/sources/setup-grafana/image-rendering/_index.md:29]: sounds perfect!
